docs/src/apireference.md

```diff
@@ -555,6 +555,26 @@ constraints of different types. There are two important concepts to distinguish:
   allow introspection into the bridge selection rationale of
   [`Bridges.LazyBridgeOptimizer`](@ref).
 
+Most bridges are added by default in [`Bridges.full_bridge_optimizer`](@ref).
+However, for technical reasons, some bridges are not added by default, for instance:
+[`Bridges.Constraint.SOCtoPSDBridge`](@ref), [`Bridges.Constraint.SOCtoNonConvexQuadBridge`](@ref)
+and [`Bridges.Constraint.RSOCtoNonConvexQuadBridge`](@ref). See the docs of those bridges
+for more information.
+
+It is possible to add those bridges and also user defined bridges, following these steps. We present an example for: [`Bridges.Constraint.SOCtoNonConvexQuadBridge`](@ref).
+
+First, define a single bridge optimizer:
```

Comment:
From the sentence above, it seems to be 2 steps that needs to be taken one after the other while it is in fact two alternative ways to do it.